From 5f7841ffffcdbdccd55f2065bef0ca16c2d79580 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?V=C3=ADt=20Ondruch?= <v.ondruch@tiscali.cz>
Date: Sun, 2 Dec 2018 22:56:50 +0100
Subject: [PATCH 1/3] Revert back to development version of .gemspec file. (#929)

This reverts commit 34a0def7d1d369a692ea5a431e127061403f6ec6, because it
seems to be completely accidental change, when the file was overridden by
Bundler or so.
---
 vagrant-libvirt.gemspec | 59 +++++++++++++----------------------------
 1 file changed, 18 insertions(+), 41 deletions(-)

diff --git a/vagrant-libvirt.gemspec b/vagrant-libvirt.gemspec
index 70610c0..dabb5f0 100644
--- a/vagrant-libvirt.gemspec
+++ b/vagrant-libvirt.gemspec
@@ -1,51 +1,28 @@
 # -*- encoding: utf-8 -*-
-# stub: vagrant-libvirt 0.0.42 ruby lib
+require File.expand_path('../lib/vagrant-libvirt/version', __FILE__)
 
 Gem::Specification.new do |s|
-  s.name = "vagrant-libvirt".freeze
-  s.version = "0.0.45"
+  s.authors       = ['Lukas Stanek','Dima Vasilets','Brian Pitts']
+  s.email         = ['ls@elostech.cz','pronix.service@gmail.com','brian@polibyte.com']
+  s.license       = 'MIT'
+  s.description   = %q{libvirt provider for Vagrant.}
+  s.summary       = %q{libvirt provider for Vagrant.}
+  s.homepage      = 'https://github.com/vagrant-libvirt/vagrant-libvirt'
 
-  s.required_rubygems_version = Gem::Requirement.new(">= 0".freeze) if s.respond_to? :required_rubygems_version=
-  s.require_paths = ["lib".freeze]
-  s.authors = ["Lukas Stanek".freeze, "Dima Vasilets".freeze, "Brian Pitts".freeze]
   s.files         = `git ls-files`.split($\)
+  s.executables   = s.files.grep(%r{^bin/}).map{ |f| File.basename(f) }
   s.test_files    = s.files.grep(%r{^(test|spec|features)/})
-  s.description = "libvirt provider for Vagrant.".freeze
-  s.email = ["ls@elostech.cz".freeze, "pronix.service@gmail.com".freeze, "brian@polibyte.com".freeze]
-  s.homepage = "https://github.com/vagrant-libvirt/vagrant-libvirt".freeze
-  s.licenses = ["MIT".freeze]
-  s.rubygems_version = "2.6.14".freeze
-  s.summary = "libvirt provider for Vagrant.".freeze
+  s.name          = 'vagrant-libvirt'
+  s.require_paths = ['lib']
+  s.version       = VagrantPlugins::ProviderLibvirt::VERSION
 
-  s.installed_by_version = "2.6.14" if s.respond_to? :installed_by_version
+  s.add_development_dependency "rspec-core", "~> 3.5.0"
+  s.add_development_dependency "rspec-expectations", "~> 3.5.0"
+  s.add_development_dependency "rspec-mocks", "~> 3.5.0"
 
-  if s.respond_to? :specification_version then
-    s.specification_version = 4
+  s.add_runtime_dependency 'fog-libvirt', '>= 0.3.0'
+  s.add_runtime_dependency 'nokogiri', '~> 1.6.0'
+  s.add_runtime_dependency 'fog-core', '~> 1.43.0'
 
-    if Gem::Version.new(Gem::VERSION) >= Gem::Version.new('1.2.0') then
-      s.add_development_dependency(%q<rspec-core>.freeze, ["~> 3.5.0"])
-      s.add_development_dependency(%q<rspec-expectations>.freeze, ["~> 3.5.0"])
-      s.add_development_dependency(%q<rspec-mocks>.freeze, ["~> 3.5.0"])
-      s.add_runtime_dependency(%q<fog-libvirt>.freeze, [">= 0.3.0"])
-      s.add_runtime_dependency(%q<nokogiri>.freeze, [">= 1.6.0"])
-      s.add_runtime_dependency(%q<fog-core>.freeze, ["~> 1.43.0"])
-      s.add_development_dependency(%q<rake>.freeze, [">= 0"])
-    else
-      s.add_dependency(%q<rspec-core>.freeze, ["~> 3.5.0"])
-      s.add_dependency(%q<rspec-expectations>.freeze, ["~> 3.5.0"])
-      s.add_dependency(%q<rspec-mocks>.freeze, ["~> 3.5.0"])
-      s.add_dependency(%q<fog-libvirt>.freeze, [">= 0.3.0"])
-      s.add_dependency(%q<nokogiri>.freeze, [">= 1.6.0"])
-      s.add_dependency(%q<fog-core>.freeze, ["~> 1.43.0"])
-      s.add_dependency(%q<rake>.freeze, [">= 0"])
-    end
-  else
-    s.add_dependency(%q<rspec-core>.freeze, ["~> 3.5.0"])
-    s.add_dependency(%q<rspec-expectations>.freeze, ["~> 3.5.0"])
-    s.add_dependency(%q<rspec-mocks>.freeze, ["~> 3.5.0"])
-    s.add_dependency(%q<fog-libvirt>.freeze, [">= 0.3.0"])
-    s.add_dependency(%q<nokogiri>.freeze, [">= 1.6.0"])
-    s.add_dependency(%q<fog-core>.freeze, ["~> 1.43.0"])
-    s.add_dependency(%q<rake>.freeze, [">= 0"])
-  end
+  s.add_development_dependency 'rake'
 end
-- 
2.20.1

